pacfish/api/adapters/Imagio_File_Converter.py

```diff
@@ -207,14 +215,14 @@ def generate_device_meta_data(self) -> dict:
         num_channels = self.pa_data.get_sizes()[0]
         for element_idx in range(num_channels): 
             detection_element_creator = DetectionElementCreator()
-            detection_element_creator.set_detector_position(np.asarray([0, 5.12 / 100 / num_channels * element_idx, 0])) # 5.12 cm probe
+            detection_element_creator.set_detector_position(np.asarray([(5.12 / 100 / num_channels) * element_idx, 0, 0])) # 5.12 cm probe
             detection_element_creator.set_detector_geometry_type("CUBOID")
             detection_element_creator.set_detector_orientation(np.asarray([0, 1, 0]))
```

Comment:
I think this should be [0, 0, 1], as the third axis is the "down" axis in the coordinate system.

![image](https://user-images.githubusercontent.com/7871475/184363994-fd59a498-4377-4e4c-b752-7f7826af8ac3.png)